### ISSUE TITLE
bug: Screenshot PNG and PDF does not follow `settings.footer.text`

### ISSUE THREAD

**Issue by gaocegege** [cncf/landscape2#491]
0.6.0 adds a new feature to customize the footer text. It works but the screenshot does not follow it. It is still `Copyright @ LF ...`

**Comment by tegioz** [comment]
Thanks for reporting this @gaocegege! We'll release version `0.6.1` a bit later today with the fix in #492 🙂

**Comment by tegioz** [comment]
BTW if the landscape you're building is publicly available, we'd love to see it @gaocegege 😇 Thanks!

**Comment by gaocegege** [comment]
Yeah sure, it's https://github.com/tensorchord/ai-infra-landscape

**Comment by gaocegege** [comment]
Thanks for the quick fix!

**Comment by tegioz** [comment]
It looks awesome, thanks! 😍
